### DISCUSSION TITLE
Share a message!

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#161] [General]
Feel free to share a message with the class below! 

You can also tag specific people by using `@___github_username__`!

My message to the class is the following:

> "Thank you for an amazing quarter! Let's keeping developing together!"

**Comment by alizezila** [comment]
Thanks for an amazing quarter! Suggestion for next time you teach the course: more Mochi. Bye y'all! It's been really fun :)

**Reply by albertkun** [reply]
@alizezila hahaha, thank you for the game changing advice!!!!

**Comment by rachan2023** [comment]
I really enjoyed this class and it was nice seeing everybody twice a week on Zoom! I can't wait to see what you guys do in the future! Also, shoutout to the people that are always hyping people up in the chat, I love the positive energy!!

**Comment by avaitt** [comment]
Loved your class so much! Thank you for giving us such a holistic overview of mapping and social justice! Also, loved Mochi!

**Comment by skarakas1** [comment]
One of the best classes I've ever taken--I am a better data ethicist and a better programmer for it. @albertkun has also facilitated a great class discussion throughout the quarter especially given that we are in zoom university, I feel like I have learned from every person in the class throughout the quarter.

**Comment by jason-muljadi** [comment]
This class has truly been a wonderful class. I am all the better because of its existence. It has taught me a myriad of things and has challenged me to be a better citizen. Thank you to my cohort and for Albert for really participating in the ways we could build on civil society.

**Comment by latishasumardy** [comment]
Thank you for such an amazing quarter! I'm so glad I took this class, even though I joined kind of late. I really learned so much from everyone, and I feel like I'm definitely going away from this class with a better understanding of how to utilize technology to help empower my community.

**Comment by rh5140** [comment]
One day, I was browsing Reddit at 2AM to procrastinate on sleeping. I saw Albert's post on r/ucla and was intrigued by the course description. I took the class on a whim — even though it doesn't fulfill any graduation requirements for me, I felt like as a computer science major, it would be important for me to take a course that emphasized ethics and how to not perpetuate harm done to marginalized communities with technology.

@albertkun  put a lot of time and care into designing this course. His lectures and the guest speakers he invites have provided me with many eye-opening insights. Before this class, I fully bought into technological solutionism, thinking that it was the role of technology to solve societal issues and change the world for the better. Under Albert's guidance, I've come to realize that that savior mindset does more harm than good, and communities should have agency over their own empowerment. I also learned to focus on accessibility to people who are less technologically literate.

Each assignment and lecture felt like it had a purpose, and I never felt like I was doing busy work! For example, we had weekly "thinking cap" discussion posts, and in my experience, responses for these types of assignments tend to be low effort. However, my classmates consistently put out thoughtful responses, and it didn't feel like a chore at all to respond to them. I've learned a lot from everyone in the class, and I'm grateful to have been exposed to so many different perspectives!

My group members @skarakas1 and @brillianz were a pleasure to work with and patient with me, for which I'm deeply grateful. I'm excited about how our project is shaping up!

This is an important disciplinary class, and since I'm only a freshman, I'll be able to stick around to tell more people to take it >:D
